ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -112,8 +112,8 @@ public extension ShadowInfo {
 public protocol Shadowable {
 
     /// The layer on which the perimeter shadow is implemented
-    var shadow1: CALayer? { get set }
+    var perimeterShadow: CALayer? { get set }
```

Comment:
Where did we get the names for perimeter and key shadows? I don't see it in the figma.

Reply:
These names align with what FURN has

Reply:
can we file a ticket to designers to update it in figma too?

Reply:
I also wouldn't take too much stock in FURN until we get designers to weigh in -- some of that code significantly predates the Fluent 2 effort.

Reply:
In FURN it's actually key and ambient, not perimeter. The Figmas I referenced all refer to the shadows as "key" and "ambient" - what do the Figmas you're looking at call them?

Reply:
We should definitely double check with designers before we merge this change in. It seems correct, but I want to make sure we're not just giving ourself more work in the future!